plugins/localize.js

```diff
@@ -70,15 +70,27 @@ module.exports.preHook = function pre(config, callback) {
     config.srcRoot = path.join(process.cwd() , 'tmp');
     destFile = path.join (config.srcRoot, 'templates', config.name + '.dust');
 
-    mkdirp(path.dirname(destFile), function(err) {
-        if (err) {
+    //check if the srcFile exists
+    fs.exists(srcFile, function (exists) {
+        var err;
+        if (!exists) {
+            err = new Error('File not found:', srcFile);
+            err.code = 'ENOENT';
             callback(err);
             return;
-        }
-        fs.createReadStream(srcFile).pipe(fs.createWriteStream(destFile));
-        callback(null, config);
+        } else {
+            mkdirp(path.dirname(destFile), function(err) {
+                if (err) {
+                    callback(err);
+                    return;
+                }
+                fs.createReadStream(srcFile).pipe(fs.createWriteStream(destFile));
```

Comment:
it can very well be just that. Will it also emit `error` if  `srcFile` path has a `dir` that is non existent... I guess it will as long as  `createReadStream` emits an `error`...